src/js/media-controller.js

```diff
@@ -625,6 +666,18 @@ const propagateMediaState = (els, stateName, val) => {
   });
 };
 
+const getStateValue = (els, stateName) => {
```

Comment:
Why was this needed?

Reply:
before we could easily check all the attribute values on media-controller but since we will not set some complex data attributes like mediarenditionlist on <media-controller> we'll have to get the values from one of the control media state receivers. cc @cjpillsbury

Reply:
Probably fine but feels a little odd to use a list of stateReceivers as the source of truth for the prior state. I wonder of we could use MediaUIStates, here or if that'd already be changed at this point. This might be a reason to move forward with state properties for media controller specifically.